xpk.py

```diff
@@ -158,6 +337,26 @@
       resources:
       - name: "{resource_type}"
         nominalQuota: {total_chips}  # Number of slices * number of chips in each slice
+  - coveredResources: ["cpu", "memory"]    
+    flavors:
+    - name: cpu-rm
+      resources:
+      - name: "cpu"
+        nominalQuota: 6
```

Comment:
How is nominalQuota for cpu/memory calculated? Should this be dynamic since we have autoscaling enabled for the pathways cpu node pools?

Reply:
Thanks for the suggestion. For now, I have set it to max-nodes (With autoscaling) * resources needed for a workload. Exploring the best set of values here. Not sure if they can be dynamic, will check.